oai_agents/common/arguments.py

```diff
@@ -83,6 +83,9 @@ def get_arguments(additional_args=[]):
     parser.add_argument("--num-dynamic-advs-per-heatmap", type=int, default=2)
     parser.add_argument("--custom-agent-ck-rate-generation", type=int)
 
+    parser.add_argument('--for-evaluation', type=bool, default=False, help="The trained agents are used for evaluating other agents. Please note that seeds and h_dim are different when agents are trained for evaluating others.)")
```

Comment:
Can you explain what 'for-evaluation' enables?

Reply:
I will update "help" there.

if for_evaluation is true, we generate population of agents for evaluation. This is currently only for population of self-play agents since we have not implemented them for other methods.

Reply:
I see! can you rename the argument to something more descriptive? for example: gen-pop-for-eval